gpu/install_gpu_driver.sh

```diff
@@ -622,13 +624,13 @@ function install_nvidia_nccl() {
     local local_tarball="${workdir}/${build_tarball}"
     local gcs_tarball="${pkg_bucket}/nvidia/nccl/${_shortname}/${build_tarball}"
 
-    if [[ "$(hostname -s)" =~ ^test && "$(nproc)" < 32 ]] ; then
+    if [[ "$(hostname -s)" =~ ^test-gpu && "$(nproc)" < 32 ]] ; then
       # when running with fewer than 32 cores, yield to in-progress build
       sleep $(( ( RANDOM % 11 ) + 10 ))
-      local output="$(${gsutil_stat_cmd} "${gcs_tarball}.building")"
+      local output="$(${gsutil_stat_cmd} "${gcs_tarball}.building"|grep '.reation.time')"
```

Comment:
limit resulting string to a single line ; the multi-line concatenation was breaking some internal tests